src/render/index.js

```diff
@@ -51,10 +56,17 @@ export default async function renderAd (selector, token, debug = false) {
 	debug && echo.info('checking that ad iframe actually rendered');
 	const ad = getTarget(`#fledge-auction-${token}`);
 	debug && echo.log('ads target:', ad);
-	if (!ad) {
+	if (!ad && hasRendered(ad)) {
```

Comment:
should this be `if( !ad || !hasRendered(ad))`? 

I'm not sure how we would not have an ad, and have it rendered.